src/test/java/uk/co/ramp/covid/simulation/population/PopulationTest.java

```diff
@@ -2,25 +2,32 @@
 
 import org.junit.Before;
 import org.junit.Test;
+import uk.co.ramp.covid.simulation.DailyStats;
 import uk.co.ramp.covid.simulation.io.ParameterReader;
-import uk.co.ramp.covid.simulation.place.Household;
+import uk.co.ramp.covid.simulation.place.*;
+import uk.co.ramp.covid.simulation.util.RNG;
 
 import java.io.IOException;
+import java.util.List;
 
 import static org.junit.Assert.*;
 
 public class PopulationTest {
 
+    Population p;
+    int populationSize;
+
     @Before
     public void setupParams() throws IOException {
-        ParameterReader.readParametersFromFile("src/test/resources/default_params.json");
+        ParameterReader.readParametersFromFile("src/test/resources/test_params.json");
```

Comment:
I think we should be testing against "default_params" here. The "test_params" (probably need renamed) is used to test the parser only.

Reply:
Ok. Had to increase the population size otherwise no hospitals are created and the model ends up hanging in method allocateCommunalPlace() in class Adult.